R/train_model.R

```diff
@@ -0,0 +1,44 @@
+# train model
+
+#' Train model
+#'
+#' @param model_formula model formula
+#' @param train_data train_data
+#' @param method method
+#' @param cv cross-validation caret scheme
+#' @param perf_metric_name performance metric name
+#' @param tune_grid tuning grid
+#' @param ntree number of trees for random forest
```

Comment:
recommend using `@inheritParams` to avoid copy-pasting documentation (.eg. `run_ml()` already has some of these)

Reply:
Done